packages/desktop-gui/src/runs/runs-list.jsx

```diff
@@ -41,11 +41,11 @@ class RunsList extends Component {
   componentDidMount () {
     this._pingApiServer()
     this._handlePolling()
-    this._getKey()
+    this._getRecordKeys()
```

Comment:
this is an unrelated rename correct?

Reply:
@Bkucera yeah somewhat, basically just being more explicit about what it's doing.